src/backend/providers/lutris/LutrisProvider.cpp

```diff
@@ -51,6 +51,17 @@ QString find_datadir()
 
     return {};
 }
+QString find_assetdir()
+{
+    const QStringList asset_roots = QStandardPaths::standardLocations(QStandardPaths::GenericCacheLocation);
+    for (const QString& asset_path : asset_roots) {
+        QString check_assetdir = asset_path + QLatin1String("/lutris/banners/");  
+        QString assetdir = asset_path + QLatin1String("/lutris/");
+        if (QFileInfo::exists(check_assetdir)) //Checks if the directory ~/.cache/lutris/banners exists
+            return assetdir;
+    }
+    return find_datadir(); //Returns the datadir instead when the above loop did not find assets under cache
+}
```

Comment:
There's no need for two different paths in the `for` loop. If we call the loop variable as `root_path`, like in `find_datadir`, then we only have to check for "assetdir = root path + /lutris/banners/".

If there are no hits, we should return an empty string. We can then handle that case in the calling function.

Reply:
Added paths `assetdir` and `banner_subdir`. The code will check for the existence of `banner_subdir` then return assetdir. If there are not hits, an empty string is returned.